cmd/utils/nodecmd/consolecmd.go

```diff
@@ -40,8 +41,11 @@ var (
 		Name:      "attach",
 		Usage:     "Start an interactive JavaScript environment (connect to node)",
 		ArgsUsage: "[endpoint]",
-		Flags:     append(ConsoleFlags, utils.DbTypeFlag, utils.SingleDBFlag, utils.NumStateTrieShardsFlag, utils.LevelDBCompressionTypeFlag, utils.DataDirFlag),
-		Category:  "CONSOLE COMMANDS",
+		Flags: append(ConsoleFlags, utils.SingleDBFlag, utils.DataDirFlag, utils.NumStateTrieShardsFlag,
```

Comment:
Is it necessary theses options related with DB for consolecmd?
@winnie-byun Please check the history and if not please remove all.